e2e/tests/replica_rebuilding.robot

```diff
@@ -36,3 +36,64 @@ Reboot Replica Node While Replica Rebuilding
         Then Wait until replica on replica node rebuilt
         And Check data is intact
     END
+
+Delete Volume Node While Replica Rebuilding
+    Given Set node-down-pod-deletion-policy to do-nothing
+    And Create deployment with rwo volume
+    And Write 2048 MB data to deployment
+
+    FOR    ${i}    IN RANGE    ${LOOP_COUNT}
+        When Delete replica on volume node to trigger replica rebuilding
+        And During replica rebuilding, delete volume node
+
+        Then Wait for volume attached and unknown
+        And Add deleted node back
+        And Wait for volume attached and healthy
+        And Wait for deployment stable
+        And Check deployment data is intact
+    END
+
+Delete Replica Node While Replica Rebuilding
+    Given Set node-down-pod-deletion-policy to do-nothing
+    And Create deployment with rwo volume
+    And Write 2048 MB data to deployment
+
+    FOR    ${i}    IN RANGE    ${LOOP_COUNT}
+        When Delete replica on replica node to trigger replica rebuilding
+        And During replica rebuilding, delete replica node
+
+        Then Wait for volume attached and degraded
+        And Add deleted node back
+        And Wait for volume attached and healthy
+        And Check deployment data is intact
+    END
+
+Force Drain Volume Node While Replica Rebuilding
+    And Create deployment with rwo volume
+    And Write 2048 MB data to deployment
+
+    FOR    ${i}    IN RANGE    ${LOOP_COUNT}
+        When Delete replica on volume node to trigger replica rebuilding
+        And Force Drain volume node
+
+        Then Wait for volume attached and degraded
+        And Uncordon volume node
+        And Wait for volume attached and healthy
+        And Wait for deployment stable
+        And Check deployment data is intact
+    END
+
+Force Drain Replica Node While Replica Rebuilding
+    And Create deployment with rwo volume
+    And Write 2048 MB data to deployment
+
+    FOR    ${i}    IN RANGE    ${LOOP_COUNT}
+        When Delete replica on replica node to trigger replica rebuilding
+        And Force Drain replica node
+
+        Then Wait for volume attached and degraded
```

Comment:
Here, the behavior is different. The volume is still attached to the volume node but becomes degraded. 

I think we should differentiate the expected result when the impacted node is volume node or replica node.

For example,

`Wait for volume attached to volume node and degraded`
`Wait for volume reattached to a new volume node and degraded`

WDYT?

Reply:
How about:

`volume still attached to the original node but degraded`
`volume reattached to another node and degraded`

Reply:
LG.

Reply:
Refined. Thank you!